app/views/claim_mailer/payment_confirmation.text.erb

```diff
@@ -2,38 +2,43 @@ Dear <%= @display_name %>,
 
 We’re paying your <%= @claim_description %>.
 
-<% if @policy == StudentLoans -%>
-This payment does not change the amount that was credited to your Student Loans Company (SLC) account in the last financial year.
-<% end -%>
-
 ^ You will receive <%=number_to_currency(@payment.net_pay) %> on or after <%= l(@payment_date) %>.
 
+<% if @payment.student_loan_repayment.present? %>
+
 # Breakdown of payment
 
-This payment is treated as part of your salary and is subject to the deductions listed below.
+This payment is treated as pay and is therefore subject to a student loan contribution, if applicable.
 
-We pay the Income Tax and National Insurance contribution (NIC) on your behalf, so these are not deducted from the amount you receive.
+* Amount you claimed for: <%= number_to_currency(@payment.award_amount) %>
+* Student loan contribution: <%= number_to_currency(@payment.student_loan_repayment) %>
+* Payment you receive: <%= number_to_currency(@payment.net_pay) %>
+
+# Contributions made to HMRC on your behalf
+
+Your payment is also subject to Income Tax and National Insurance (NI) contributions, which we pay to HMRC on your behalf. This does not affect the amount you receive.
+
+* Income Tax: <%= number_to_currency(@payment.tax) %>
+* Employee NI: <%= number_to_currency(@payment.employers_national_insurance) %>
+* Gross amount paid by us: <%= number_to_currency(@payment.gross_value) %>
```

Comment:
Should this be:

```suggestion
* Gross amount paid by us: <%= number_to_currency(@payment.gross_pay) %>
```

i.e. their gross pay, not the "gross_value" that DfE have paid.

Reply:
Ahh yes, I thought this might be the case. I got confused by the line under it and misreading employee as employer! Will fix